timesketch/lib/analyzers/feature_extraction.py

```diff
@@ -229,7 +266,46 @@ def extract_feature(self, name, config):
                 continue
 
             event_counter += 1
-            event.add_attributes({store_as: result[0]})
+            store_type = event.source.get(store_as)
```

Comment:
I would say that this is not a `store_type` but rather the original value... 

so rather say

`old_value = eve....store_as)` or `current_value = ....` or something more descriptive here.